userprofile/templates/userprofile/login.html

```diff
@@ -20,84 +20,130 @@ <h1>{% trans "Login" %}</h1>
 	<div id="login-box-row" class="row">
 
 		{% if shibboleth_login %}
-			<div class="col-sm-6 col-md-3 login-box">
-				<div class="thumbnail">
-					<div class="caption">
+			<section aria-label="{{ shibboleth_title_text|safe }} login" class="col-sm-6 col-md-3 login-box">
```

Comment:
I don't think the `safe` filter is safe to use here since unescaped citation marks would break the HTML attribute. Is `shibboleth_title_text` translated in the view? I think so, and then appending "login" to the end does not work for every language.